### PR TITLE
Completed CSS Flexbox

### PR DESCRIPTION
**Pull request by SharjeelSIqbal**
CSS Flex Display
CSS Flex Wrap
CSS Justify Content
CSS Align Items

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```
Comment (codingcodymiller) [inline css-flexbox/align-items.html]: This should be applied to the `card-container`, not the deck. `flex-basis` is one of the strange flexbox properties that is actually applied to the child elements, not the parent element.

File: css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (codingcodymiller) [inline css-flexbox/flex-wrap.html]: This should be applied to the `card-container`, not the deck. `flex-basis` is one of the strange flexbox properties that is actually applied to the child elements, not the parent element.

File: css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (codingcodymiller) [inline css-flexbox/justify-content.html]: This should be applied to the `card-container`, not the deck. `flex-basis` is one of the strange flexbox properties that is actually applied to the child elements, not the parent element.

File: css-flexbox/justify-content.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    .card-container {
+      justify-content: center;
```
Comment (codingcodymiller) [inline css-flexbox/justify-content.html]: This should be applied to the deck, not the `card-container`.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-flexbox/align-items.html [hunk 5]
```diff
@@ -11,12 +11,15 @@
...
       display: flex;
-      flex-basis: calc(100% / 3);
-      min-height: 100vh;
       justify-content: center;
       align-items: center;
     }
 
+    .card-container {
+      flex-basis: calc(100% / 3);
+      min-height: 100vh;
```
Comment (codingcodymiller) [inline css-flexbox/align-items.html]: This line should be on `#card-deck`. Did your output look correct after making this change? 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-flexbox/align-items.html (hunk 5 above)
Comment (SharjeelSIqbal) [reply css-flexbox/align-items.html]: I didn't do a double take once I made the changes thought it might magically work for some reason.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**